hail/src/test/scala/is/hail/expr/ir/BlockMatrixIRSuite.scala

```diff
@@ -119,6 +119,37 @@ class BlockMatrixIRSuite extends HailSuite {
     }
   }
 
+  @Test def testBlockMatrixFilter() {
```

Comment:
are these actually using the relational strategy? 

Er, just confirmed, yes they are 👍